site/docs/v1/tech/email-templates/email-templates.adoc

```diff
@@ -59,587 +59,8 @@ This field supports replacement variables.
 
```

Comment:
Line 18 needs to be removed as it displays "Broken Link"

Reply:
Good catch!